openlibrary/templates/book_providers/cita_press_read_button.html

```diff
@@ -0,0 +1,21 @@
+$def with(cita_press_id)
+
+<div class="cta-button-group">
+  <a
+      href="https://citapress.org/$cita_press_id"
+      title="$_('Read eBook from Cita Press')"
+      class="cta-btn cta-btn--available cta-btn--read cta-btn--external cta-btn--cita-press"
+      target="_blank"
+       aria-haspopup="true"
+       aria-controls="cita-press-toast"
+  >$_('Read')</a>
+</div>
+
+$if render_once('cita-press-toast'):
+  <div class="toast toast--book-provider" data-toast-trigger=".cta-btn--openstax" id="cita-press-toast" style="display:none">
+    <div class="toast__body">
+      $:_('This book is available from <a href="https://citapress.org/">Cita Press</a>. TODO')
```

Comment:
@mekarpeles If you can fill out this TODO, I think it should be good to go! Take a look at the other provides for an idea of wording.